python/batch-predictor/requirements.txt

```diff
@@ -1,6 +1,7 @@
-pyspark==2.4.5
+findspark
+pyspark==3.0.1
 mlflow>=1.2.0,<=1.23.0
 cloudpickle==2.0.0
-pyarrow==0.14.1
-protobuf>=3.0,<4.0.0dev
+pyarrow>=0.14.1
```

Comment:
What happen if users uses older version of pyarrow?

Reply:
I guess it should still work depending on the python versions. Anyway 0.14.1 was released in 2019 and only supports up till python 3.7. Loosening this restriction is to support python 3.8 to 3.10. I will add a upper limit for testing purpose, as per suggested.